pom.xml

```diff
@@ -45,11 +46,10 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>${version.junit}</version>
```

Comment:
Looks suspicious?

Reply:
Do we inherit that from parent?

Reply:
> The managed version is 4.12 The artifact is managed in org.wildfly.core:wildfly-core-parent:3.0.8.Final

And we did not specify `version.junit` in this pom...

Reply:
Ah OK, thanks.